packages/cli/src/ui/hooks/atCommandProcessor.ts

```diff
@@ -459,7 +460,7 @@ export async function handleAtCommand({
     toolCallDisplay = {
       callId: `client-read-${userMessageTimestamp}`,
       name: readManyFilesTool.displayName,
-      description: readManyFilesTool.getDescription(toolArgs),
+      description: `Read files: ${pathSpecsToRead.join(', ')}`,
```

Comment:
why isn't this
invocation.getDescription()?

Reply:
I've updated the CL to support this. TLDR: because we may not have an invocation at this point. However, I've hoisted the argument so in the case where we do have a `ToolInvocation` we can use it, and if not we just returned a canned error description.

However, given this is an error path, I'm actually not 100% sure what the description should be in this case? Should we be saying we've read files when in fact there was some error?